src/scene/constants.js

```diff
@@ -287,8 +287,15 @@ export const SHADOW_PCF5 = 4;
  */
 export const SHADOW_PCF1 = 5;
 
+/**
+ * Render depth as color for PCSS software filtering.
+ *
+ * @type {number}
+ */
+export const SHADOW_PCSS = 6;
+
 // non-public: number of supported depth shadow modes
-export const SHADOW_COUNT = 6;
+export const SHADOW_COUNT = 7;
```

Comment:
just looking at the codebase, this is no longer needed, I missed its removal when I refactored it, could you please remove SHADOW_COUNT completely.

Reply:
Fixed